service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/service/EncounterReportBundleService.java

```diff
@@ -76,11 +77,12 @@ private static void addEntry(Bundle bundle, Resource resource) {
     public Bundle createEncounterBundle(POCDMT000002UK01ClinicalDocument1 clinicalDocument, ItkReportHeader header) throws XmlException {
         Bundle bundle = createBundle(clinicalDocument);
 
+        MessageHeader messageHeader = messageHeaderService.createMessageHeader(header);
```

Comment:
This `messageHeader` variable could be used in line 97 in `addEntry` method

Reply:
good catch, fixed